### DISCUSSION TITLE
Tesla Fleet API

### DISCUSSION THREAD

**Discussion by DomiSanNo1** [pkuehnel/TeslaSolarCharger#1091] [Q&A]
Is it possible to read out the new Fleet API Token after creating it in TSC? My other Tokens e.g. In OpenHab doesnt work anymore since i‘ve created the Fleet API Token.

**Comment by pkuehnel** [comment]
The Token is in the TeslaSolarCharger.db file which is an SQLite Database. But this has nothing to do with your token in OpenHab not working anymore. This is just a coincidence. Tesla is disabling the old Api the last days for all accounts, so very likely your OpenHub extension has not shifted to use the fleet API which is no longer supported by Tesla. The TSC Token won't help as Tesla disabled the old API for your account. The new token has nothing to do with the old ones.

**Reply by DomiSanNo1** [reply]
Ok, thx. In case of this i have to wait for a workaround in the oh binding.
